packages/core/src/common/classes.ts

```diff
@@ -83,7 +83,7 @@ export const RTL = `${NS}-rtl`;
 // layout utilities
 /**
  * @see https://developer.mozilla.org/en-US/docs/Web/CSS/Containing_block#identifying_the_containing_block
- * @deprecated this is no longer needed for ContextMenu2, will be removed in v4.0
+ * @deprecated this is no longer needed for ContextMenu2, will be removed in v5.0
```

Comment:
this is actually removed in v4.0, so this line should not change. see https://github.com/palantir/blueprint/wiki/Blueprint-4.0